src/adapters/lido/tests/snapshots/ethereum.prices.json

```diff
@@ -59,8 +59,8 @@
               "symbol": "stETH",
               "decimals": 18,
               "type": "underlying",
-              "underlyingRateRaw": "1143460902465163371n",
-              "underlyingRate": "1.143460902465163371",
+              "underlyingRateRaw": "1143913215373500178n",
+              "underlyingRate": "1.143913215373500178",
```

Comment:
This will be fixed with my PR. Currently, the wsEth prices are not taking into consideration the block number.